.circleci/config.yml

```diff
@@ -306,7 +313,11 @@ workflows:
           <<: *release-tags
           requires:
             - export-package
+
+  snapshot-bump:
+    when:
+      not:
+        equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
```

Comment:
This isn't really needed just yet here (until we have added release trains) but I thought I would add it.